stateth/config.go

```diff
@@ -0,0 +1,49 @@
+// Copyright 2018 The go-ethereum Authors
+// This file is part of the go-ethereum library.
+//
+// The go-ethereum library is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Lesser General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// The go-ethereum library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with the go-ethereum library. If not, see <http://www.gnu.org/licenses/>.
+
+package stateth
+
+import (
+	"os"
+)
+
+// DefaultConfig contains default settings for the stateth.
+var DefaultConfig = Config{
+	DockerPrefix:     "stateth",
+	DashboardsFolder: os.Getenv("GOPATH") + "/src/github.com/ethereum/go-ethereum/stateth/grafana_dashboards",
+	GrafanaPort:      3000,
+	InfluxDBPort:     8086,
+	Rm:               false,
```

Comment:
With `rm == false`, if we just kill Geth, after having it started with `geth --dashboard --metrics --metrics.influxdb.export`, we won't be able to start it back again. I think this is fine for a small executable like `stateth`, which if you use, you should have an idea of what it's doing, but I am not convinced this is an OK behaviour for Geth.

Probably it'd be best if we have this set to `true` for Geth, so that it always starts up correctly.

What do you think?